docs/package/gluon-scheduled-domain-switch.rst

```diff
@@ -21,18 +21,26 @@ domain_switch : optional (needed for domains to switch)
         - amount of time without reachable gateway to switch unconditionally
     switch_time :
         - UNIX epoch after which domain will be switched
-    connection_check_targets :
-        - array of IPv6 addresses which are probed to determine if the node is
-	  connected to the mesh
+
```

Comment:
The documentation should be part of your own package, not of the scheduled domain switch.

Reply:
I just noticed, that the documentation wasnt clear there, should I make a separate PR for this docs change of the sheduled-domain-switch package?

Reply:
Ah OK, so I'll add an extra docs page for just the check-connection package then

Reply:
not an extra page, but moved to generic site conf docs